docs/admin/access.rst

```diff
@@ -273,30 +275,30 @@ granted by site-wide or per-project groups by adding another custom group.
 
 ..
 
-  3. Add users you wish to give the permissions to into the group.
+  3. Add users you wish to give the permissions to into this group.
 
 As you can see, permissions management this way is powerful,
 but can be quite a tedious job. You can't
-delegate it to another user unless granting superuser permissions.
+delegate it to another user, unless granting superuser permissions.
 
 .. _auth-model:
 
-Users, roles, groups and permissions
-++++++++++++++++++++++++++++++++++++
+Users, roles, groups, and permissions
++++++++++++++++++++++++++++++++++++++
 
 The authentication models consist of several objects:
 
 `Permission`
-    Individual permissions defined by Weblate. Permissions can not be
+    Individual permission defined by Weblate. Permissions cannot be
     assigned to users. This can only be done through assignment of roles.
 `Role`
     A Role defines a set of permissions. This allows reuse of these sets in
```

Comment:
should be "role"

Reply:
Wanna make the suggestion to correct it?

Reply:
I don't know the markup used. I like to not touch that stuff other than in my own PRs.
Somehow the ability to make suggestions goes away if things have been edited before, or it has something to do with review. Works now though.

Reply:
I understand, OK, I will do the correction

Reply:
Done

Reply:
done